src/bench.c

```diff
@@ -747,6 +747,27 @@ int benchmark_all(void)
 /* Silently skip formats for which we have no tests, unless forced */
 		if (!format->params.tests && format != fmt_list)
 			continue;
+		
+/* Format disabled in john.conf */
+		if (cfg_get_bool(SECTION_DISABLED, SUBSECTION_FORMATS,
```

Comment:
Very strange: Even though line 750 looks fine here, and this supposedly is the very commit that was used, it now has whitespace noise (two tabs).